bin/openstudio_meta

```diff
@@ -23,9 +23,10 @@ Signal.trap('INT') { abort }
 require 'logger'
 $logger = ::Logger.new(STDOUT)
 $logger.level = ::Logger::WARN
-ENV['GEM_PATH'] = File.absolute_path(File.join(__FILE__, './../../gems/')).to_s + File::PATH_SEPARATOR + File.absolute_path(File.join(__FILE__, './../../gems/bundler/gems')).to_s
```

Comment:
I honestly don't know the different between GEM_PATH and GEM_HOME, but in practice it seems that as along as GEM_PATH is set, everything seems to work.